spec/fixtures/bibliography/book.bib

```diff
@@ -10,5 +10,5 @@ @book{http://zotero.org/groups/1051392/items/TXXUJDG2
 	keywords = {sg012cr8689},
 	annote = { KAW},
 	annote = {CCC MS 470 [p. 9, and referred to (without MS number) throughout]reproductions from CCC MS 470, ff. 125-146, in the appendix},
```

Comment:
You're removing the multiple `annote` keys in other fixtures. Do you want to do the same for this fixture, or do you have a spec in mind for handling this case?

Reply:
Nope! Just a typo 😅